GDJS/Runtime/howler-sound-manager/howler-sound-manager.js

```diff
@@ -420,12 +533,30 @@ gdjs.HowlerSoundManager.prototype.preloadAudio = function (
 
   for (var file in files) {
     if (files.hasOwnProperty(file)) {
-      var httpRequest = new XMLHttpRequest();
-      httpRequest.addEventListener('load', onLoad);
-      httpRequest.addEventListener('error', onLoad);
-      httpRequest.addEventListener('abort', onLoad);
-      httpRequest.open('GET', file);
-      httpRequest.send();
+      if(files[file].preloadAsSound) {
+        // Load as sound
+        var sound = new Howl({
+          src: [file],
+          preload: true,
+          onload: onLoad,
```

Comment:
If a file is both `preloadAsSound` and `preloadAsMusic`, onLoad will be called twice, which will make the loading inconsistent. You probably need to think of another way of ensuring both are loaded before validating the load of this resource as a whole.